backend/test/initialization/initialization_test.go

```diff
@@ -38,7 +38,7 @@ func (s *InitializationTest) SetupTest() {
 		return
 	}
 
-	err := test.WaitForReady(*namespace, *initializeTimeout)
```

Comment:
Actually, `WaitForReady` does no longer depend on any namespace [since now it invokes `localhost:8888`](https://github.com/kubeflow/pipelines/blob/998a2aa6995738beb05dea4c280a94c5708d3922/backend/test/v2/test_utils.go#L41).